src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { StatusFilter } from './helpers/StatusFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const [todoFilter, setTodoFilter] = useState<StatusFilter>(StatusFilter.All);
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
```

Comment:
can we create function like `selectTodo` and there put whole `todo` in our state.

```jsx
  const selectTodo = (todo: Todo) => {
    setSelectedTodo(todo);
  };
  ```

Reply:
Sorry, didn`t see that